apps/01PigLatin.js

```diff
@@ -7,10 +7,92 @@ prompt.start();
 
 function pigLatin(word) {
 
-    // Your code here
+  // function pigLatin(word) {
+  //   //for a simple word
+  //   //this assigns the first letter of the word to the variable firstLetter
+  //     var firstLetter = word [0];
+  //   //this takes the first letter c and moves it to the end and adds ay to it
+  //     return word.replace (firstLetter, '' ) + firstLetter + 'ay';
+  //added in class for notes
 
-}
 
+    // Concept Check
+    //
+    // Please explain why we are using String.prototype.toLowerCase() here.
+    // For more info:
+    // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase
+    //
+    // Your Answer:
+    // When people are prompted to put in a word they may not always put in a lowercase letter or an uppercase letter. Because of this when you are looking for those letters later one you would have to code for lowercase and uppercase. To go around that you can add the .toLowerCase to the word which changes whaterver is put into lowercase letters. It simplifies the code.
+   word  = word.toLowerCase();
+
+   //find a more complex
+   // Concept Check
+   //
+   // What is the significance of setting vowelIndex to -1?
+   // This might help:
+   // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/lastIndexOf
+   //
+   // Your Answer: The reason it is set to -1 is because it is the value if there is not a vowel. This set of code uses arrays to check the word for the vowels. Arrays are always 0 - length of word. If there is a vowel it will come back a positive integer. If there is no vowel when the vowelIndex is set to -1 the answer would be -1. If this number was a positive number you would run into issues with longer words. This allows different lengths of words to be placed in the app and the code still works the same.
```

Comment:
Let's talk over this answer together in class. You've got the general idea right, just need to be clearer.